payload-command/render/renderassets/assets.go

```diff
@@ -177,6 +177,31 @@ func ClusterProfile(clusterProfile string) FileContentsPredicate {
 	}
 }
 
+// BootstrapRequiredCRD returns a predicate for LoadFilesRecursively that filters manifests
+// based on whether they are marked as being required for bootstrap or not.
+func BootstrapRequiredCRD() FileContentsPredicate {
+	return func(manifest []byte) (bool, error) {
+		uncastObj, _, err := codecs.UniversalDecoder().Decode(manifest, nil, &unstructured.Unstructured{})
+		if err != nil {
+			panic(fmt.Errorf("unable to decode: %w", err))
+		}
+
+		isBootstrapCRD := uncastObj.(*unstructured.Unstructured).GetAnnotations()["release.openshift.io/bootstrap-required"]
```

Comment:
don't we only provide manifests in our repo that are for bootstrapping?  Why do we need to filter here versus at the bash script that does the copy?